server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -79,3 +85,44 @@ export function generateUser(): IUser {
 
     return randomUser;
 }
+
+/**
+ * Verifies the storage token claims and calls riddler to validate the token.
+ */
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+export function verifyStorageToken(tenantManager: ITenantManager, config: Provider): RequestHandler {
+    return (request, res, next) => {
+        const maxTokenLifetimeSec = config.get("auth:maxTokenLifetimeSec") as number;
+        const isTokenExpiryEnabled = config.get("auth:enableTokenExpiration") as boolean;
+        const authorizationHeader = request.header("Authorization");
+        if (!authorizationHeader) {
+            return res.status(401).json("Authorization header is missing.");
+        }
+        const regex = /Basic (.+)/;
+        const tokenMatch = regex.exec(authorizationHeader);
+        if (!tokenMatch || !tokenMatch[1]) {
+            return res.status(401).json("Missing access token.");
+        }
+        const token = tokenMatch[1];
+        const tenantId = getParam(request.params, "tenantId");
+        const documentId = getParam(request.params, "id") || request.body.id;
+        if (!tenantId || !documentId) {
+            return res.status(401).json("TenantId or DocumentId is missing in the access token.");
+        }
+        const claims = validateTokenClaims(token, documentId, tenantId, maxTokenLifetimeSec, isTokenExpiryEnabled);
+        if (!claims) {
+            return res.status(401).json("Invalid access token.");
+        }
+        console.log(`token to verify: ${token}`);
```

Comment:
Remove this

Reply:
oops! yes, I was debugging something here. Will remove it.

I'm working on fixing the unit tests. So will push the fixes soon.